charmcraft.yaml

```diff
@@ -68,6 +68,28 @@ config:
       description: "Whether or not csi-*plugin-provisioner deployments use host-networking"
       type: boolean
 
+    csidriver-name-formatter:
+      default: "{name}"
+      description: |
+        Formatter for the 2 CSIDrivers managed by this charm:
+          - rbd.csi.ceph.com
+          - cephfs.csi.ceph.com
+
+        The formatter is a string that can contain the following placeholders:
+          - {name}             - original name from the manifest
+          - {app}              - the name of the juju application
+          - {namespace}        - the charm configured namespace
+
+        Example:
+          juju config ceph-csi csidriver-name-formatter="{name}.{app}"
```

Comment:
maybe this isn't the best example???

Which would you rather see in your cluster:
*`ceph-csi-alt.rbd.csi.ceph.com`
*`rbd.csi.ceph.com.ceph-csi-alt`